BaragonService/src/main/java/com/hubspot/baragon/service/managers/ElbManager.java

```diff
@@ -162,6 +169,7 @@ private void registerNewInstances(List<LoadBalancerDescription> elbs, BaragonGro
           LOG.info(String.format("Registered instances %s with ELB %s", request.getInstances(), request.getLoadBalancerName()));
         } catch (AmazonClientException e) {
           LOG.error("Could not register %s with elb %s due to error %s", request.getInstances(), request.getLoadBalancerName(), e);
+          exceptionNotifier.notify(e, ImmutableMap.of("elb", request.getLoadBalancerName(), "toAdd", request.getInstances().toString()));
```

Comment:
hmm, that could work too. I know in Singularity we do it in this style (LOG.error then notify). @tpetr was there an initial reason we did it that way for Singularity, or would a logback appender be more appropriate here?